tests/networkpolicy_test.go

```diff
@@ -44,7 +44,6 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		tests.PanicOnError(err)
 
 		tests.SkipIfUseFlannel(virtClient)
-		tests.SkipIfNotUseNetworkPolicy(virtClient)
```

Comment:
Please mention in this commit why this can be removed. I honestly do not understand why.

Reply:
@qinqon found out it actually works on k8s cluster (but flakey)
the original commit that added the skip 2 years ago along with the tests as they are on master,
said that back then it was supported on openshift only,
so its fine to mention its safe to run it on U/S and the skip isnt needed right ?

Reply:
done

Reply:
OK, I understand. 

Do you happen to know what changed - i.e. how are we able to run this now on k8s clusters ? If so, add it to the commit msg.